lib/activerecord_spanner_adapter/base.rb

```diff
@@ -117,6 +125,7 @@ def self._create_grpc_values_for_insert metadata, values
       serialized_values = []
       columns = []
       values.each_pair do |k, v|
+        v = unwrap_attribute v
```

Comment:
Is this a change that is needed for 7.x, or was this an existing bug in the current implementation?

Reply:
This is to catch up with this ActiveRecord 7.x change.
https://github.com/rails/rails/commit/6ee96a8f42d6b13bffd46342248f447d9f289288
With this change merged into ActiveRecord 7.0, we will receive `ActiveModel::Attribute` instances for the return value of `#attributes_with_values` so they need to be unwrapped. In ActiveRecord 6.x we get the values itself, hence [the conditional](https://github.com/googleapis/ruby-spanner-activerecord/pull/170/commits/dd650979d4d83e3d096b82333c64ed077c28b875#diff-2b0d1b3234f56c21e27aa0c68126687695874ba8603f41bd571c7847397b8988R91-R95) here.